code/modules/mob/living/carbon/human/human_examine.dm

```diff
@@ -64,7 +64,7 @@
 		if(!article_override)
 			article = starts_with_vowel(displayed_species) ? "an" : "a"
 
-		msg += ", [article]<b><font color='[examine_color]'> [lowertext(displayed_species)]</font></b>!"
+		msg += ", [article]<b><font color='[examine_color]'> [lowertext(displayed_species)]</font></b>, <b>[p_are(FALSE)] are of [height] and of [physique].</b>.!"
```

Comment:
```suggestion
		msg += ", [article]<b><font color='[examine_color]'> [lowertext(displayed_species)]</font></b>, <b>[p_are(FALSE)] of [height] and of [physique].</b>.!"
```

Reply:
CMON DGL THIS IS THE ONE, 5th TIMES THE CHARMMMM